pyproject.toml

```diff
@@ -71,6 +70,11 @@ include = ["frescobaldi*"]
 [tool.setuptools.dynamic.version]
 attr = "frescobaldi.appinfo.version"
 
+# .mo files are not tracked in git
+# `tox -e mo-generate` needs to be run before `python -m build`
+[tool.setuptools.package-data]
+"*" = ["*.mo"]
```

Comment:
Given that in #1898 we do not install from a full git checkout (using the tarball that we mirror and users verify with a manifest, git is inconvenient there), seems this currently (only) installs the .mo and still skips all the other files.